### PR TITLE
EZP-31774: Main fixes after ux review

### PR DESCRIPTION
**Pull request by lucasOsti**
jira ref: https://jira.ez.no/browse/EZP-31774

### CODE REVIEW

File: src/bundle/Resources/public/scss/_custom.scss [hunk 1]
```diff
@@ -12,16 +12,19 @@ $ibexa-black: #131C26;
 $ibexa-color-primary: #DB0032;
 $ibexa-color-primary-pale: #F3E4E7;
 $ibexa-color-primary-light: #E75476;
+$ibexa-color-maximum-black: #222222;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss:15]: `maximum-black` doesn't sound like a proper name

File: src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
     text-decoration: none;
-    transition: background 0.3s $ez-admin-transition;
-    background: $ez-white;
+    transition: all 0.3s $ez-admin-transition;
+    background: $ibexa-white;
     border: calculateRem(1px) solid rgba(0, 0, 0, 0.1);
-    box-shadow: 0 calculateRem(2px) calculateRem(10px) calculateRem(0px) rgba(0, 0, 0, 0.15);
-
+    transition: all $ez-admin-transition;
+
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss:12]:
```suggestion

```

File: src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss [hunk 3]
```diff
@@ -40,28 +39,25 @@
...
-        display: -webkit-box;
-        -webkit-line-clamp: 2;
-        -webkit-box-orient: vertical;
-        overflow: hidden;
-        line-height: calculateRem(15px);
-        font-size: calculateRem(12px);
-        margin: calculateRem(8px) calculateRem(5px);
+        display: flex;
+        align-items: center;
+        width: 248px;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss:58]: calcrem

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_custom.scss (hunk 1 above)
Comment (GrabowskiM) [reply src/bundle/Resources/public/scss/_custom.scss:15]: ibexa-color-blackest-black :P

File: src/bundle/Resources/public/scss/_user-menu.scss [hunk 4]
```diff
@@ -7,32 +7,32 @@
...
-                transform: translate(3px, -50%) rotate(-135deg);
-            }
-
-            &:after {
-                transform: translate(-3px, -50%) rotate(135deg);
+                width: 0; 
+                height: 0; 
+                border-left: calculateRem(5px) solid transparent;
+                border-right: calculateRem(5px) solid transparent;
+
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_user-menu.scss:25]: empty line

File: src/bundle/Resources/public/scss/mixins/_modals.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+        .ez-icon {
+            opacity: 1;
+        }
+    }
+}
+
+@mixin modal-footer() {
+    background: $ibexa-color-base-pale;
+        padding: $modal-footer-padding;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/mixins/_modals.scss:43]: bad indent

File: src/bundle/Resources/views/themes/admin/account/notifications/list.html.twig [hunk 6]
```diff
@@ -27,11 +27,11 @@
     </tbody>
 </table>
 {% if pager.haveToPaginate %}
-    <div class="row justify-content-center align-items-center mb-2 mt-3">
+    <div class="row justify-content-center align-items-center ez-pagination__spacing">
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/account/notifications/list.html.twig:30]: I think there's no base class `ez-pagination` anywhere?

**Review by GrabowskiM — CHANGES_REQUESTED** [review]
Almost in all occurences there is class `ez-pagination__spacing` without parent class, ony here it's done correctly: https://github.com/ezsystems/ezplatform-admin-ui/pull/1455/files#diff-e04f99099a5cc47cacd5fcd8956643f7R70

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss [hunk 7]
```diff
@@ -1,22 +1,21 @@
...
-    color: $ez-black;
+    color: $ibexa-color-font;
     text-decoration: none;
-    transition: background 0.3s $ez-admin-transition;
-    background: $ez-white;
+    transition: all 0.3s $ez-admin-transition;
+    background: $ibexa-white;
     border: calculateRem(1px) solid rgba(0, 0, 0, 0.1);
-    box-shadow: 0 calculateRem(2px) calculateRem(10px) calculateRem(0px) rgba(0, 0, 0, 0.15);
+    transition: all $ez-admin-transition;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss:11]: transition is defined above

File: src/bundle/Resources/public/scss/ui/modules/sub-items-list/_table.view.item.scss [hunk 8]
```diff
@@ -107,16 +108,25 @@
 
     &__priority-wrapper {
         position: relative;
+
+        &--focused {
+            .ez-input {
+                border-radius: $ibexa-border-radius;
+                border: 2px solid $ibexa-color-primary;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/sub-items-list/_table.view.item.scss:115]: calcRem

**Review by dew326 — CHANGES_REQUESTED** [review]

**Pull request merged**